browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -15,89 +15,68 @@
  */
 
 import UIKit
-import OktaOidc
+import WebAuthenticationUI
 
 final class WelcomeViewController: UIViewController {
-
-    var oktaOidc: OktaOidc?
-    var stateManager: OktaOidcStateManager?
+    lazy var auth = WebAuthentication.shared
     
+    @IBOutlet weak var signInButton: UIButton!
     @IBOutlet weak var ephemeralSwitch: UISwitch!
+    @IBOutlet weak var clientIdLabel: UILabel!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationItem.hidesBackButton = true
-        
-        if #available(iOS 13.0, *) {
-            // We'll allow the switch to be shown
+        if let clientId = auth?.signInFlow.client.configuration.clientId {
+            self.clientIdLabel.text = "clientId: \(clientId)"
```

Comment:
```suggestion
            clientIdLabel.text = "Client ID: \(clientId)"
```

Reply:
Alternatively you can do the same thing I did in the other samples, and place two labels into an HStack